src/components/problem/styledProblem.js

```diff
@@ -31,6 +66,19 @@ export const Title = styled.h2`
   line-height: 52px;
   letter-spacing: 0em;
   margin-bottom: 20px;
+
+  @media( max-width: 1023px ) {
+    max-width: 647px;
+    margin: 0 auto 40px auto;
+    font-size: 38px;
+    line-height: 45px;
+    text-align: center;
+  }
+  @media( max-width: 767px ) {
+    margin-bottom: 32px;
+    font-size: 28px;
+    line-height: 32px;
+  }
```

Comment:
Title and subtitle should be 343px and 346 px at 375 screen width.   I'd probably do this with a percentage and just meet in the middle or something (both at 345px)

Reply:
I'm going to leave this as is because there is basically no difference when you add that width: 343px to each text. It shifts the text over like 5 pixels. Not noticeable to the naked eye.